amansachdeva13/kpifire
# If new pr created with link

*Include label if *S*Q*L changes were made or an API needs to be run*

### 1. URL to corresponding KPI Fire Task:
https://dev.kpifire.com/#/project/1018/tasks/51097 (Task 1)
https://dev.kpifire.com/#/project/1017/tasks/51087 (Task 1)

### 2. Is it a bug or new feature/logic? If bug, why was it not working?
UI Imp

### 3. Steps for Success Test:

![delUser](https://github.com/KPIFire/kpifire/assets/10572402/ca4aa40a-161d-467e-996e-e0ca3fd26a1d)

1. go to settings - Users - Invite Users
2. delete user invite 
3. title should be Delete User Invite
4. Are you sure you want to delete [user name]?

### 4. Regression Testing: Which areas will/could be affected?:
Y/N/NA : Mobile APIs impacted?

### 5. for PM: How should this item be listed in release notes?
